### PR TITLE
Add callout about human data and security

### PR DESCRIPTION
**Pull request by nicjar**
This is in response to https://github.com/datacarpentry/cloud-genomics/issues/73
Added a general paragraph about human data and the need to be aware of any ethical and legal requirements when working with human genomic data.

### CODE REVIEW

File: _episodes/04-which-cloud.md [hunk 1]
```diff
@@ -181,6 +181,21 @@ my university HPCC for running the full dataset.
 > - How many times will you need to run this pipeline?
 {: .challenge}
 
+> ## Human genomic data & Security
+> 
+> Note that if you are working with human genomics data there might be ethical and legal 
+> considerations that affects your choice of cloud resources to use. The terms of use, and/or
```
Comment (ACharbonneau) [inline _episodes/04-which-cloud.md]: affect

File: _episodes/04-which-cloud.md [hunk 2]
```diff
@@ -181,6 +181,21 @@ my university HPCC for running the full dataset.
...
+> 
+> Note that if you are working with human genomics data there might be ethical and legal 
+> considerations that affects your choice of cloud resources to use. The terms of use, and/or 
+> the legislation under which you are handling the genomic data, might impose heightened information 
+> security measures for the computing environment in which you intend to process it. This is a too broad 
+> topic to discuss in detail here, but in general terms you should think through the technical and 
+> procedural measures needed to ensure that the confidentiality and integrity of the human data you work 
+> with is not breached. If there are laws that govern these issues in the jurisdiction in which you work, 
+> be sure that the cloud service provider you use can certify that they support the necessary measures. 
+> Also note that there might exist restrictions of use of cloud service providers that operate in other
```
Comment (ACharbonneau) [inline _episodes/04-which-cloud.md]: restrictions for use

**Review by ACharbonneau — CHANGES_REQUESTED** [review]
Thanks for this! Could you change the two little grammar things and I'll merge it in.

**Comment by nicjar** [comment]
Changes done!

**Pull request merged**